src/pages/ProductDetail/FavoriteBtn.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+const FavoriteBtn = ({
+  children,
+  color,
+  size,
+  outline,
+  fullWidth,
+  ...rest
+}) => {
+  return (
+    <StyledButton
+      color={color}
+      size={size}
+      outline={outline}
+      fullWidth={fullWidth}
+      {...rest}
+    >
+      {children}
+    </StyledButton>
+  );
+};
+export default FavoriteBtn;
+
+FavoriteBtn.defaultProps = {
+  color: 'white',
+  size: 'medium',
+};
+
+const colorStyles = css`
+  ${({ theme, color }) => {
+    const selected = theme.palette[color];
+    return css`
+      background: ${selected};
+      ${props =>
+        props.outline &&
+        css`
+          color: ${selected};
+          background: none;
+          border: 1px solid ${selected};
+        `}
+    `;
+  }}
+`;
+
+const sizes = {
```

Comment:
```suggestion
const SIZES = {
```

상수인것 같네요, 상수임이 표현되도록 네이밍 컨벤션 잘 적용해주세요!

Reply:
수정완료!